### PR TITLE
Documentation - Add design diagrams

### PR DESCRIPTION
**Pull request by SharonHart**
Add design diagrams to desgin.md and to each service description

### CODE REVIEW

File: presidio-anonymizer/README.MD [hunk 1]
```diff
@@ -5,6 +5,8 @@
 The Presidio anonymizer is a Python based module for anonymizing detected PII text
 entities with desired values.
 
+![Anonymizer Design](../docs/assets/anonymizer-design.png)
```
Comment (SharonHart) [inline presidio-anonymizer/README.MD:8]: Both here, and for the image redactor, should be change to '../assets/analyzer-design.png' as for the analyzer, after the docs will move the docs dir

Comment (navalev) [reply presidio-anonymizer/README.MD:8]: why cant this be done now?

**Review by omri374 — APPROVED** [review]

File: presidio-anonymizer/README.MD (hunk 1 above)
Comment (SharonHart) [reply presidio-anonymizer/README.MD:8]: It can, understood from @shiranr in the daily that shes taking it?

File: docs/design.md [hunk 2]
```diff
@@ -1,5 +1,11 @@
 # Presidio design
 
-## Table of contents
+## Analyzer
+[![Analyzer Design](assets/analyzer-design.png)](analyzer/index.md)
```
Comment (SharonHart) [inline docs/design.md:4]: @omri374 Should I use absolute path here as well? How is the linking handled in mkdocs?

**Review by navalev — APPROVED** [review]

**Pull request merged**